sdk/webpubsub/Azure.Messaging.WebPubSub/api/Azure.Messaging.WebPubSub.netstandard2.0.cs

```diff
@@ -1,5 +1,10 @@
 namespace Azure.Messaging.WebPubSub
 {
+    public static partial class SearchFilter
```

Comment:
The name is too generic, consider adding a prefix

Reply:
Referenced https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/src/SearchFilter.cs#L19